src/__tests__/counterComment.test.js

```diff
@@ -0,0 +1,12 @@
+import countComment from "../counterComment";
+
+describe("testing count comment", () => {
+
+
+test("Counting comments", () => {
+   let commentList = [];
+   commentList.push("BA");
+    
+    expect(countComment(commentList)).toBe(1) 
+})
+})
```

Comment:
Great job testing the counter 👍 . As the first reviewer said, kindly add more tests to cover all possible edge cases, I noticed you have not implemented this.

[OPTIONAL] I think you can mock the HTML so as to get an accurate test. ☺️ 🟢